content/en/account_management/billing/custom_metrics.md

```diff
@@ -5,68 +5,199 @@ aliases:
+
+To obtain the temperature in Florida, you can simply recombine the custom metrics of:
+
+* `temperature:{country:USA, state:Florida, city:Orlando}`
+* `temperature{country:USA, state:Florida, city:Miami}`
+
+**Note**: Reordering tag values doesn’t add uniqueness. The following combinations are the same custom metric:
+
+*  `temperature{country:USA, state:Florida, city:Miami}`
+*  `temperature{state:Florida, city:Miami, country:USA}`
+
+[1]: /developers/metrics/types/?tab=count#metric-submission-types
+[2]: /developers/metrics/types/?tab=rate#metric-submission-types
+[3]: /developers/metrics/types/?tab=gauge#metric-submission-types
+{{% /tab %}}
+{{% tab "Histogram" %}}
+
+**A HISTOGRAM metric generates by default five custom metrics** for each unique combination of metric name and tag values to support the Agent-side aggregations `max`, `median`, `avg`, `95pc`, and `count`. [Learn more about HISTOGRAM metric type][1].
+
+The number of unique tag value combinations submitted for a HISTOGRAM metric with this tagging scheme is **four**:
+
+* `host:A`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:400`
+* `host:B`, `endpoint:Y`, `status:200`
+
+By default, the Agent generates five custom metrics for each of the original four unique tag value combinations to account [for each Agent-side aggregations enabled][2]: `max`, `median`, `avg`, `95pc`, and `count`. Consequently, `request.Latency` reports a total of **4*5 = 20 custom metrics**.
+
+**Notes**: Adding aggregations to your HISTOGRAM metrics increases the number of distinct custom metrics reported. Removing aggregations decreases the number of custom metrics reported.
 
-Integrations limited to 350 custom metrics by default:
+* Configure which aggregation you want to send to Datadog with the `histogram_aggregates` parameter in your [datadog.yaml configuration file][3]. By default, only `max`, `median`, `avg`, and `count` aggregations are sent out to Datadog. `sum` and `min` are also available if desired.
+* Configure which percentile aggregation you want to send to Datadog with the `histogram_percentiles` parameter in your [datadog.yaml configuration file][3]. By default, only the `95pc` percentile is sent out to Datadog.
 
-* [ActiveMQ XML][4]
-* [Go-Expvar][5]
+[1]: /developers/metrics/types/?tab=histogram
+[2]: /developers/metrics/types/?tab=histogram#metric-submission-types
+[3]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+{{% /tab %}}
+{{% tab "Distribution" %}}
 
-Integrations with no default limit:
+**A DISTRIBUTION metric generates by default five custom metrics** for each unique combination of metric name and tag values to represent the global statistical distribution of values. These five custom metrics represent server-side aggregations of `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
```

Comment:
**A DISTRIBUTION metric generates by default five custom metrics for each unique combination of metric name and tag values** to represent the global statistical distribution of values. These five custom metrics represent server-side aggregations of `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].